pyproject.toml

```
@@ -41,7 +41,7 @@ ethereum = ["eth_account>=0.4.0"]
 polkadot = ["substrate-interface==1.3.4"]
 cosmos = ["cosmospy==6.0.0"]
 solana = ["pynacl==1.5.0", "base58==2.1.1"]
-tezos = ["pynacl==1.5.0", "aleph-pytezos==0.1.0"]
+tezos = ["pynacl==1.5.0", "aleph-pytezos@git+https://github.com/aleph-im/aleph-pytezos.git@main"]
```

We cannot publish `aleph-client` on PyPI if all dependencies are not on PyPI.
```suggestion
tezos = ["pynacl==1.5.0", "aleph-pytezos==3.13.4"]
```

---

Ah, forget to push, this was already done, sowwy